tests/test_docstrings/test_numpy.py

```diff
@@ -367,6 +373,21 @@ def test_returns_section(parse_numpy: ParserType) -> None:
         DocstringReturn(name="", annotation="bool", description="Some kind\nof flag."),
     )
 
+    assert_element_equal(
+        sections[0].value[2],
+        DocstringReturn(name="", annotation=None, description="Name only"),
```

Comment:
Shouldn't it pick the name here?

```suggestion
        DocstringReturn(name="x", annotation=None, description="Name only"),
```

Reply:
Ahhhh, assert_element_equal doesn't test name 😬. Going through now to see how to work it in